model_utils/managers.py

```diff
@@ -260,11 +261,16 @@ def delete(self):
         self.update(is_removed=True)
 
 
-class SoftDeletableManager(models.Manager):
+class SoftDeletableQuerySet(SoftDeletableQuerySetMixin, QuerySet):
+    pass
+
+
+class SoftDeletableManagerMixin(object):
     """
     Manager that limits the queryset by default to show only not removed
     instances of model.
     """
+    use_for_related_fields = True
```

Comment:
Correct, but since all the others Managers in that file have it, I've added it as it looked consistent with the project (also probably because this is what I'm doing in my project).

But you're right it's unrelated, I'll remove it.